src/Instance/HoldingsList/Holding/HoldingContainer.js

```diff
@@ -102,18 +112,12 @@ const HoldingContainer = ({
   ...rest
 }) => {
   const onViewHolding = useCallback(() => {
-    history.push({
-      pathname: `/inventory/view/${instance.id}/${holding.id}`,
-      search: location.search,
-    });
+    window.location.href = `/inventory/view/${instance.id}/${holding.id}${location.search}`;
```

Comment:
this reloads the page

Reply:
Good catch, @Dmytro-Melnyshyn!

Reply:
You are right, @Dmytro-Melnyshyn, this is what @OleksandrHladchenko1 supposed to do when used `window.location.href` instead of `history.push`. The page needs to be refreshed because user's affiliation changes, tenant is updated, new permissions are fetched so it is similar to switching affiliation as it is done in `ui-consortia-settings`

Reply:
Okay, my comment just vanished for some reason.
So the page is supposed to be reloaded because the affiliation is changed, new tenant permissions are fetched, and as a result, other modules are available for the user. The same logic is implemented in `ui-consortia-settings` module when switching affiliation.